wiki/Interface/en.md

```diff
@@ -307,7 +307,7 @@ This is your online leaderboard. You can go here by scrolling down from the resu
    2. `Accuracy`: Your [average accuracy](/wiki/Accuracy#accuracy) over all maps you have played.
```

Comment:
```suggestion
   2. `Accuracy`: Your average [accuracy](/wiki/Accuracy#accuracy) over all maps you have played.
```

Reply:
This is what the old version looks like. I haven't changed it, but I will edit it